### PR TITLE
Jn user profile

### PR DESCRIPTION
**Pull request by jniewiel**

### CODE REVIEW

File: app/views/comments/_form.html.erb [hunk 1]
```diff
@@ -11,22 +11,13 @@
...
-  <div>
-    <%= form.label :author_id, style: "display: block" %>
-    <%= form.text_field :author_id %>
-  </div>
-
-  <div>
-    <%= form.label :photo_id, style: "display: block" %>
-    <%= form.text_field :photo_id %>
-  </div>
+  <%= form.hidden_field :photo_id %>
```
Comment (aborov) [inline app/views/comments/_form.html.erb:14]: Great use of hidden params, so they get passed without user having to filling it out

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          <div class="card-body">
+            <h5 class="card-title"><%= photo.owner.username %></h5>
+            <p class="card-text"><%= photo.caption %></p>
+          </div>
+          <ul class="list-group list-group-flush">
+            <% photo.comments.each do |comment| %>
+              <li class="list-group-item">
+                <div class="d-flex">
+                  <div class="flex-shrink-0">
+                    <img src="..." alt="...">
```
Comment (aborov) [inline app/views/users/show.html.erb:22]: Would you consider commenting out this `img` placeholder for now?

File: app/views/comments/_form.html.erb [hunk 3]
```diff
@@ -11,22 +11,13 @@
...
 
-  <div>
-    <%= form.label :body, style: "display: block" %>
-    <%= form.text_area :body %>
+  <div class="mb-3">
+    <%= form.text_area :body, class: "form-control" %>
```
Comment (aborov) [inline app/views/comments/_form.html.erb:17]: Cool Bootstrapping here!

File: config/routes.rb [hunk 4]
```diff
@@ -8,6 +8,8 @@
   resources :follow_requests
   resources :likes
   resources :photos
+
+  get "/:username" => "users#show", as: :user
```
Comment (aborov) [inline config/routes.rb:12]: I love that you keep your user-specific route at the bottom

**Review by aborov — APPROVED** [review]
Great work! Looking forward to the next part!

File: app/controllers/comments_controller.rb
Comment (aizatibraimova) [inline app/controllers/comments_controller.rb:1]: I love that your code is easy to follow.

File: app/controllers/users_controller.rb
Comment (aizatibraimova) [inline app/controllers/users_controller.rb:1]: Well done on handling potential edge cases with ActiveRecord exceptions.

**Review by aizatibraimova — APPROVED** [review]

File: app/controllers/comments_controller.rb [hunk 5]
```diff
@@ -58,13 +59,14 @@ def destroy
...
-      params.require(:comment).permit(:author_id, :photo_id, :body)
-    end
+  # Use callbacks to share common setup or constraints between actions.
+  def set_comment
+    @comment = Comment.find(params[:id])
+  end
+
+  # Only allow a list of trusted parameters through.
+  def comment_params
+    params.require(:comment).permit(:author_id, :photo_id, :body)
```
Comment (JoseMDO) [inline app/controllers/comments_controller.rb:70]: nice!

File: app/views/users/show.html.erb (hunk 2 above)
Comment (JoseMDO) [inline app/views/users/show.html.erb:22]: Yes great point, you could also add default image to users who don't have an image :)

File: config/routes.rb (hunk 4 above)
Comment (JoseMDO) [inline config/routes.rb:12]: Agree!

**Review by JoseMDO — APPROVED** [review]
Great job Jan!

**Pull request merged**